src/Avalonia.DesignerSupport/Remote/HtmlTransport/HtmlTransport.cs

```diff
@@ -262,5 +260,87 @@ private void OnSignalTransportException(IAvaloniaRemoteTransportConnection arg1,
             _onException?.Invoke(this, ex);
         }
         #endregion
+
+        private void ProcessingReceiveMessage(string message)
+        {
+            var parts = message.Split(':');
+            switch (parts[0])
+            {
+                case "frame-received":
+                    {
+                        _onMessage?.Invoke(
+                            this,
+                            new FrameReceivedMessage { SequenceId = long.Parse(parts[1]) });
+                        break;
+                    }
+                case "pointer-released":
+                    {
+                        _onMessage?.Invoke(
+                            this,
+                            new InputProtocol.PointerReleasedEventMessage
+                            {
+                                Modifiers = ParseInputModifiers(parts[1]),
+                                X = double.Parse(parts[2]),
```

Comment:
This will fail on several non-english locales, use CultureInfo.InvariantCulture

Reply:
Oops, thanks! Fixed it